translate/tests/test_asttoirvisitor.py

```diff
@@ -1,12 +1,9 @@
-#!/usr/bin/env python3
-import unittest
```

Comment:
I thought you wanted this to be a python unittest?

Reply:
pytest